### PR TITLE
curl, open, powershell, wget: remove "view in older versions of `tldr`"

### PR DESCRIPTION
**Pull request by acuteenvy**
This example is only in those three pages, all others have the updated option from the current specification (`-p`/`--platform`).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by kbdharun** [comment]
Update: the Node client still seems to be requiring this option. I will open a PR to update the behaviour as suggested in https://github.com/tldr-pages/tldr/pull/10848#discussion_r1346967482.

**Comment by reinhart1010** [comment]
Hi, as the original author of these pages, I still prefer to keep it while waiting for the official Node.js client to be fixed, more on #8297.

**Comment by kbdharun** [comment]
> Hi, as the original author of these pages, I still prefer to keep it while waiting for the official Node.js client to be fixed, more on https://github.com/tldr-pages/tldr/pull/8297.

Agreed, Thanks for the info (on the other PR, I forgot the Node client doesn't use platform options)

I have marked this PR as a draft, post the change is done. I will open it back for review.

**Comment by kbdharun** [comment]
Update: Created a PR for it https://github.com/tldr-pages/tldr-node-client/pull/421, let's see how it goes.

**Comment by acuteenvy** [comment]
The stalebot was removed a couple months ago.

**Comment by kbdharun** [comment]
Marking this PR as open for review since the OS option has been changed to platform in our Node client.

Now, we need to remove this line across the pages.

**Review by gutjuri — APPROVED** [review]

File: pages/common/open.md
Comment (kant) [inline pages/common/open.md:1]: LGTM

**Review by kbdharun — APPROVED** [review]

File: pages/common/powershell.md
Comment (kant) [inline pages/common/powershell.md:1]: LGTM

File: pages/windows/curl.md
Comment (kant) [inline pages/windows/curl.md:1]: LGTM

File: pages/common/powershell.md
Comment (kant) [inline pages/common/powershell.md:1]: LGTM

File: pages/windows/wget.md
Comment (kant) [inline pages/windows/wget.md:1]: LGTM

**Pull request merged**